app/wasm_test.go

```diff
@@ -233,3 +234,19 @@ func TestMsgTypeBlacklistMessenger_DispatchMsg(t *testing.T) {
 			}),
 	).Run(t)
 }
+
+func TestNewWasmAppModuleBasicOverride(t *testing.T) {
+	uploader := authtypes.NewModuleAddress("governance")
```

Comment:
```suggestion
	uploader := authtypes.NewModuleAddress("gov")
```
module name is `gov`, for consistency

Reply:
```suggestion
	uploader := authtypes.NewModuleAddress("allowed to upload")
```